crates/uv/tests/pip_sync.rs

```diff
@@ -4208,7 +4209,7 @@ fn require_hashes_unnamed() -> Result<()> {
     Ok(())
 }
 
-/// We allow `--require-hashes` for editables, as long as no dependencies are included.
+/// We disallow `--require-hashes` for editables.
```

Comment:
Let's say i have some local packages i want to install as editables, and pypi deps i want to install with `--require-hashes`, does this mean i can't do this anymore?

Reply:
Yeah, can’t do this anymore. I can special-case it if we want? It’s consistent with pip so I wasn’t sure.